pkg/kv/kvserver/store_create_replica.go

```diff
@@ -106,64 +167,40 @@ func (s *Store) tryGetOrCreateReplica(
 	creatingReplica *roachpb.ReplicaDescriptor,
 ) (_ *Replica, created bool, _ error) {
 	// The common case: look up an existing (initialized) replica.
```

Comment:
Does the "initialized" here mean something else? Probably it means initialization in the regular sense, not in "uninitialized/initialized replica" sense.

Reply:
Yeah, I think it just means "not unloaded". Better to remove this.